redis/cluster.py

```diff
@@ -956,7 +956,7 @@ def determine_slot(self, *args):
             determine what slots to map the command to; or, if the keys don't
             all map to the same key slot.
         """
-        command = args[0]
+        command = args[0].upper()
```

Comment:
It's better to do this in this line:
`if command in ("EVAL", "EVALSHA"):`
because in other cases we call `self._get_command_keys(*args)`, that needs the lower command.

Reply:
you mean `if command.upper() in ("EVAL", "EVALSHA"):` ?

Reply:
@yangbodong22011 Yes

Reply:
done